core/tx_pool_test.go

```diff
@@ -87,6 +97,22 @@ func pricedDataTransaction(nonce uint64, gaslimit uint64, gasprice *big.Int, key
 	return tx
 }
 
+func dynamicFeeTx(nonce uint64, gaslimit uint64, gasFee *big.Int, tip *big.Int, key *ecdsa.PrivateKey) *types.Transaction {
+	tx := types.NewTx(&types.DynamicFeeTx{
```

Comment:
You can use `types.SignNewTx(..)` instead

Reply:
right, done